examples/nixnet/can-signal-single-point-output.py

```diff
@@ -0,0 +1,125 @@
+r""" Write Signal Data.
+
+ This example writes a signal value when a keyboard character is pressed.
+ This is used to demonstrate a signal single point output session. 
+ This example uses hardcoded signal names that use the NIXNET_example database. 
+
+The gRPC API is built from the C API. NI-XNET documentation is installed with the driver at:
+  C:\Users\Public\Documents\National Instruments\NI-XNET\Documentation\NI-XNET Manual.chm
+Getting Started:
+
+To run this example, install "NI-XNET Driver" on the server machine:
+  https://www.ni.com/en-in/support/downloads/drivers/download.ni-xnet.html
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Refer to the NI XNET gRPC Wiki for the latest C Function Reference:
+  https://github.com/ni/grpc-device/wiki/NI-XNET-C-Function-Reference
+
+ Running from command line:
+Server machine's IP address and port number can be passed as separate command line
+arguments.
+  > python can-signal-single-point-output.py <server_address> <port_number>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763"
+"""
+
+import sys
+
+import getch
```

Comment:
Consider: Do we really need an external dependency for `q`-to-quit? There are several examples that use `ctrl+c` and maybe one that uses esc. Will one of those work for you instead? Can you require <enter> for a newline instead of a character? Can you just run 10 times without user input?

Also, if you add this, you need to add it to validate_examples as a dependency to install.

Reply:
Yeah, so we are going ahead with running this 10 times without user input and printing output. Removing the `getch` dependency.